### PR TITLE
DOCS-4234 update

### PR DESCRIPTION
**Pull request by george-hoffer**

### CODE REVIEW

File: modules/ROOT/pages/mq-apis.adoc [hunk 1]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+
+You can get an immediate snapshot of how many messages are in the queue and how many are in-flight (see xref::mq-understanding.adoc[Anypoint MQ Terminology] for what in-flight messages are). This data is updated in near real-time. The Statistics API can fetch real time statistics for multiple queues, and returns a comma separated list. 
+
+`/queues?destinationIds={queue1},{queue2}`
+
+*Sample Request and Response:*
+
+[source,example,linenums]
+----
+http://localhost:8080/api/v1/organizations/<org_ID>/environments/<env_ID>/regions/us-east-1
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this URL should be changed to https://anypoint.mulesoft.com/

File: modules/ROOT/pages/mq-apis.adoc [hunk 2]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+
+This API can be used to fetch historic stats for a queue, however it does not get updated in real-time, and there is an approximate latency of 5 to 10 minutes before data is published. The retention period of historical data is 15 months. Also, to keep the volume of data under control, historical data is stored at reduced levels of granularity. Data points with a period of less than 60 seconds are available for 3 hours, one minute data points are available for 15 days, five minute data points for 63 days, and one hour data points for 15 months.
+
+`/queues/{queueId}?startDate=...&endDate=...&period=...`
+
+*Sample Request and Response:*
+
+[source,example,linenums]
+----
+http://localhost:8080/api/v1/organizations/<org_ID>/environments/<env_ID>/regions/us-east-1
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this URL should be changed to https://anypoint.mulesoft.com/

File: modules/ROOT/pages/mq-apis.adoc [hunk 3]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+
+Retention policies are same as in the previous <<Historic Statistics for a Queue>> section.
+
+`/exchanges/{exchangeId}?startDate=...&endDate=...&period=...`
+
+*Sample Request and Response:*
+
+[source,example,linenums]
+----
+https://localhost:8080/api/v1/organizations/<org_ID>/environments/<env_ID>/regions/us-east-1
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this URL should be changed to https://anypoint.mulesoft.com/

File: modules/ROOT/pages/mq-apis.adoc [hunk 4]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+        }
+     ]
+}
+----
+
+Note that period is an integer here. It's the granularity of datapoints in seconds.
+
+=== Usage Metrics by Environment 
+
+Anypoint MQ uses the Mule Analytics service to store usage metric data. The Stats API can be used to sum up the usage for Anypoint MQ across all environments for a particular organization. This is not real time and has a latency.
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this need not be part of public docs. Remove the part about Analytics service.

File: modules/ROOT/pages/mq-apis.adoc [hunk 5]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+
+`/environments/{environmentId}?startDate=...&endDate=...&period=...`
+
+*Sample Request and Response:*
+
+[source,example,linenums]
+----
+http://localhost:8080/api/v1  /organizations/<org_ID>/environments/<env_ID>
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this URL should be changed to https://anypoint.mulesoft.com/

File: modules/ROOT/pages/mq-apis.adoc [hunk 6]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+        "timestamp": "2018-11-08T00:00Z",
+        "apiRequestCount": 0,
+        "messageReceiptCount": 0,
+        "billableUnitCount": 0,
+        "messageByteCount": 0
+    }
+]
+----
+
+The `period` duration supports these values:
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: These are part of our raml docs already. And we need not add such api specifications on public tutorial. I think we can remove this.

File: modules/ROOT/pages/mq-apis.adoc [hunk 7]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+`3hours` +
+`1day` +
+`7days` +
+`1month` +
+`3months`
+
+
+=== Usage Metrics by Organization
+
+Anypoint MQ uses the Mule Analytics service to store usage metric data for an organization. The Stats API can be used to sum up the usage for Anypoint MQ across the master business organization:
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: remove info about analytics service

File: modules/ROOT/pages/mq-apis.adoc [hunk 8]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+
+`/organizations/{organizationId}?startDate=...&endDate=...&period=...`
+
+*Sample Request and Response:*
+
+[source,example,linenums]
+----
+http://localhost:8080/api/v1
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: this URL should be changed to https://anypoint.mulesoft.com/

File: modules/ROOT/pages/mq-apis.adoc [hunk 9]
```diff
@@ -172,14 +180,173 @@ This returns output similar to:
...
+        "timestamp": "2018-11-08T00:00Z",
+        "apiRequestCount": 0,
+        "messageReceiptCount": 0,
+        "billableUnitCount": 0,
+        "messageByteCount": 0
+    }
+]
+----
+
+The `period` duration supports these values:
```
Comment (jjayakumaran) [inline modules/ROOT/pages/mq-apis.adoc]: need not be added on public documentation

**Review by jjayakumaran — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/mq-apis.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed throughout

File: modules/ROOT/pages/mq-apis.adoc (hunk 4 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed

File: modules/ROOT/pages/mq-apis.adoc (hunk 5 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed

File: modules/ROOT/pages/mq-apis.adoc (hunk 7 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed

File: modules/ROOT/pages/mq-apis.adoc (hunk 8 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed

File: modules/ROOT/pages/mq-apis.adoc (hunk 9 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Removed

File: modules/ROOT/pages/mq-apis.adoc (hunk 6 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Removed

File: modules/ROOT/pages/mq-apis.adoc [hunk 10]
```diff
@@ -142,19 +143,26 @@ image::mq-rest-broker-portal.png[]
...
+* Usage metrics - View your entire organization's usage of Anypoint MQ for billing purposes:
+** Usage per environment
+** Usage per organization
+* Browse statistics for queues and message exchanges  
+* View the number of messages currently in a queue
 
-You can get the access token from the Anypoint Platform user. The access token is also known as the Bearer.
+=== Use the Stats API
+
+To use this API, you can get the access token from the Anypoint Platform user. The access token is also known as the Bearer.
```
Comment (steveCluster) [inline modules/ROOT/pages/mq-apis.adoc]: can you add this here: To access the Anypoint MQ Stats API, you must first authenticate using the Anypoint Access Management API

The part where it says Anypoint Access Management API should be a link to this url: https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/

File: modules/ROOT/pages/mq-apis.adoc [hunk 11]
```diff
@@ -172,14 +180,149 @@ This returns output similar to:
 }
 ----
 
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-stats/[Stats API] - Perform statistical analysis of queue performance.
+https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/anypoint-mq-stats/[Statistics API] - Perform statistical analysis of queue performance.
```
Comment (steveCluster) [inline modules/ROOT/pages/mq-apis.adoc]: Instead of using actual org and env IDs can you put in placeholders like ORGANIZATION_ID and ENVIRONMENT_ID

**Review by jjayakumaran — APPROVED** [review]
Please fix Steve's comments before merge. Thanks

File: modules/ROOT/pages/mq-apis.adoc (hunk 10 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Fixed as requested

File: modules/ROOT/pages/mq-apis.adoc (hunk 11 above)
Comment (george-hoffer) [reply modules/ROOT/pages/mq-apis.adoc]: Done.

**Pull request merged**